apb.yml

```diff
@@ -1,7 +1,7 @@
 version: 1.0
 name: unifiedpush-apb
 description: AeroGear UnifiedPush Server
-bindable: True
+bindable: False
```

Comment:
@matzew This turned out to be the reason for the error in the service instance. I don't think this APB needs to be bindable or do you see a reason?

Reply:
correct - not needed - ATM !